### PR TITLE
Remove double support section and match new design

### LINKED ISSUE #78: Remove bottom `supporters`, move brains to the upper section and match the design on Figma
We've decided to keep it simple when it comes to showcasing logos of entities and for that reason, we're only showing entities that are currently contributing to the SV2 either financially by funding devs or providing other sort of help (dev help, testing, etc). We're removing entities that have "said they'll do something with SV2 in the future, until they do something, as it's getting hard to determine contributions level and may be unfair to entities that have been doing work for months, some years.

For that reason, it's critical that we update our existing supporters section to match 1:1 the Figma design

Figma https://www.figma.com/file/H42PrGnniKFtbfqHYBGevj/Stratum?node-id=0%3A1

#### How the section looks now
Screenshot 2022-09-09 at 09 23 39

#### How the section should look when addressed with a PR

Screenshot 2022-09-09 at 09 20 08

### PR DESCRIPTION
**Pull request by secondl1ght**
Fixes #78

### CODE REVIEW

**Comment by pavlenex** [comment]
@secondl1ght Nice, there's something I found though, the sparkles are  kinda messed up

Preview on Firefox
Screenshot 2022-09-09 at 18 44 45

Preview on Safari
Screenshot 2022-09-09 at 18 44 59

Can you figure out what went wrong there?

File: src/.vuepress/theme/layouts/HomeLayout.vue [hunk 1]
```diff
@@ -106,6 +106,11 @@
 
       <!-- Road map -->
       <div class="flex relative z-20 flex-col items-center px-8">
+        <img
+          src="/assets/deco-cyan.png"
```
Comment (SeraBocca) [inline src/.vuepress/theme/layouts/HomeLayout.vue:110]: here you can just change .png to .svg to make it work

**Comment by SeraBocca** [comment]
> @secondl1ght Nice, there's something I found though, the sparkles are kinda messed up
> 
> Preview on Firefox Screenshot 2022-09-09 at 18 44 45
> 
> Preview on Safari Screenshot 2022-09-09 at 18 44 59
> 
> Can you figure out what went wrong there?
> 
> Seems linkcheck caught it?
> 
> ```
> ├─BROKEN─ http://localhost:8080/assets/deco-cyan.png (HTTP_404)
> [6](https://github.com/stratum-mining/stratumprotocol.org/runs/8274609243?check_suite_focus=true#step:6:7)
> ```

yep that's because i changed format in a previous PR https://github.com/stratum-mining/stratumprotocol.org/pull/85/files#r967291270

**Review by pavlenex — DISMISSED** [review]
LGTM now, will let @SeraBocca do the final check and merge if all good.

**Review by pavlenex — APPROVED** [review]

**Pull request merged**